tox.ini

```diff
@@ -22,7 +22,9 @@ commands =
 
 [pytest]
 addopts = --tb=short --durations=10
-markers = integration
+markers =
+    integration
```

Comment:
I think it would be ok to just run a full sweep of `integration` tests for redirector testing

Reply:
The `integration` marker appears to be used for something else.  Specifically, [just this test](https://github.com/dandi/dandi-cli/blob/5a4f24c518b9fda2e2343afce5f2fc88fc7c20da/dandi/tests/test_organize.py#L105).